components/Modal/CreateImageUpload.js

```diff
@@ -442,88 +496,297 @@ class CreateImageUploadModal extends React.Component {
       )
     };
 
-    const uploadAuth = provider => ({
+    const awsUploadAuth = {
       name: "Authentication",
       component: (
         <React.Fragment>
           <Text className="help-block cc-c-form__required-text">
             <FormattedMessage defaultMessage="All fields are required." />
           </Text>
-          {providerSettings[provider] !== undefined && (
-            <Form isHorizontal>
-              {Object.keys(providerSettings[provider].auth).map(key => (
-                <FormGroup
-                  label={formatMessage({
-                    id: `${provider}-auth`,
-                    defaultMessage: providerSettings[provider].auth[key].displayText
-                  })}
-                  fieldId={key}
-                  key={key}
+          <Form isHorizontal className="cc-m-wide-label">
+            <div className="pf-c-form__group">
+              <div className="pf-c-form__label pf-m-no-padding-top pf-l-flex pf-u-display-flex pf-m-justify-content-flex-start pf-m-nowrap">
+                <label htmlFor="access-key-id-input" className="pf-l-flex__item">
+                  <span className="pf-c-form__label-text">
+                    <FormattedMessage defaultMessage="Access key ID" />
+                  </span>
+                  <span className="pf-c-form__label-required" aria-hidden="true">
+                    &#42;
+                  </span>
+                </label>
+                <Popover
+                  id="access-key-popover"
+                  bodyContent={
+                    <FormattedMessage
+                      defaultMessage="You can create and find existing Access key IDs on the {iam} page in the AWS console."
+                      values={{
+                        iam: <strong>Identity and Access Management (IAM)</strong>
+                      }}
+                    />
+                  }
+                  aria-label="access key id help"
                 >
-                  <TextInput
-                    value={this.state.uploadSettings[key] || ""}
-                    type={providerSettings[provider].auth[key].isPassword ? "password" : "text"}
-                    id={key}
-                    key={key}
-                    onChange={this.setUploadSettings}
-                    isRequired
-                  />
-                </FormGroup>
-              ))}
-            </Form>
-          )}
+                  <Button variant="plain" aria-label="access key id help">
```

Comment:
Just to be save, can we make acronyms upper-case. I _think_ most assistive technology is pretty decent at identifying acronyms when all caps are used.